Makefile

```diff
@@ -227,6 +227,24 @@ CALICO_VERSION ?= v3.26.3
 crs-calico: ## Generates crs manifests for Calico.
 	curl -o templates/crs/cni/calico.yaml https://raw.githubusercontent.com/projectcalico/calico/$(CALICO_VERSION)/manifests/calico.yaml
 
+METALLB_VERSION ?= main
+FRR_K8S_DIR = metallb/charts/metallb/charts/frr-k8s/templates
+.PHONY: crs-metallb
+crs-metallb: ## Generates crs manifests for MetalLB.
+	#@ TODO: temporarily pull metallb from main, update when release happens
+	@rm -rf metallb
+	@git clone https://github.com/metallb/metallb/ metallb/ -n -q -b main --single-branch --filter=blob:none
```

Comment:
Is there a reason you're cloning the repo?

You can use the helm repo
```
helm repo add metallb https://metallb.github.io/metallb
helm install metallb metallb/metallb
```

Reply:
Unfortunately the route import feature is broken in metallb 0.13.12, so I've been talking with the MetalLB guys and they guaranteed to me that it's fixed in the next release and that release will be out soon.

In fact the release just dropped: https://metallb.universe.tf/release-notes/
I'll be removing most of this.

Reply:
Done